### PR TITLE
fix: format check stage in the CI

### PR DESCRIPTION
**Pull request by kelkarajay**
This PR fixes the CI workflow. The format checks had been moved to an independent workflow (https://github.com/ory/kratos/blob/master/.github/workflows/format.yml) to be triggered for PR's and master builds.

## Related issue(s)

Build failure on master

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -83,8 +83,6 @@ jobs:
         uses: sonatype-nexus-community/nancy-github-action@v1.0.2
       - run: npm install
         name: Install node deps
-      - name: Run prettier
```
Comment (aeneasr) [inline .github/workflows/ci.yaml:86]: Is the formatting now checked somewhere else?

Comment (kelkarajay) [reply .github/workflows/ci.yaml:86]: Yep, https://github.com/ory/kratos/blob/master/.github/workflows/format.yml

Comment (kelkarajay) [reply .github/workflows/ci.yaml:86]: We can also rollback https://github.com/ory/kratos/commit/20fdfe8ddfe8f2922a8fa6c544319ebd8fa5d524 if the `format` check needs to be kept as a pre-req for tests

Comment (aeneasr) [reply .github/workflows/ci.yaml:86]: 👍

**Review by aeneasr — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (kelkarajay) [reply .github/workflows/ci.yaml:86]: Take a look now please

**Review by aeneasr — APPROVED** [review]

**Pull request merged**